server/pkg/config/trdl.go

```diff
@@ -19,6 +19,7 @@ type Trdl struct {
 	DockerImage    string   `yaml:"dockerImage,omitempty"`
 	DockerImageOld string   `yaml:"docker_image,omitempty"` // legacy
 	Commands       []string `yaml:"commands,omitempty"`
+	BuilderImage   string   `yaml:"builderImage,omitempty"`
```

Comment:
It is not developer space and should be configured by an administrator. I would suggest not supporting a custom builder image for now and instead adding new endpoints as needed in the future.